vignettes/qenv.Rmd

```diff
@@ -26,18 +26,16 @@ print(empty_qenv)
 
 ### `qenv` basic usage
 
-The `eval_code()` function executes code within a `qenv` environment, yielding a new `qenv` object as the output. 
+The `eval_code()` function executes code within a `qenv` environment, yielding a new `qenv` object as the output.
 
 ```{r}
-library(magrittr)
-
 # evaluate code in qenv
 my_qenv <- eval_code(empty_qenv, "x <- 2")
 print(my_qenv)
 get_env(my_qenv)
 
 
-q1 <- eval_code(my_qenv, "y <- x * 2") %>% eval_code("z <- y * 2")
+q1 <- eval_code(my_qenv, "y <- x * 2") |> eval_code("z <- y * 2")
```

Comment:
`|>` require R >= 4.1. I suggest to replace `|>` or `%>%` with normal nested R calls

Reply:
good idea, I will replace all `|>` occurences as you suggested.